frontend/src/app/aethel/aethel.component.html

```diff
@@ -0,0 +1,119 @@
+<h2 class="title is-2" i18n>Æthel</h2>
+
+<p class="field" i18n>
+    Welcome to Æthel. Search for a word or lemma in the input field below to
+    start.
+</p>
+
+<form class="form" [formGroup]="form" (ngSubmit)="search()">
+    <div class="field">
+        <label for="aethel-input" class="label">Search:</label>
+        <div class="aethel-input-container">
+            <input
+                id="aethel-input"
+                class="input"
+                [class.is-danger]="form.touched && form.invalid"
+                type="text"
+                formControlName="aethelInput"
+                placeholder="Enter a word or lemma..."
+                placeholder-i18n
+            />
+            <button
+                type="submit"
+                class="button is-primary"
+                [class.is-loading]="loading$ | async"
+            >
+                <span i18n>Search</span>
+            </button>
+        </div>
+        @if (form.touched && form.invalid) {
+        <p class="is-size-7 has-text-danger" i18n>Please enter at least three characters.</p>
+        }
+    </div>
+</form>
+
+@if (submitted | async) {
+<div class="table-container">
+    <p-table
+        [value]="rows"
+        dataKey="key"
+        [paginator]="true"
+        [rows]="10"
+        [rowsPerPageOptions]="[10, 25, 50]"
+        [showCurrentPageReport]="true"
+        currentPageReportTemplate="Showing {first} to {last} of {totalRecords} entries"
+        currentPageReportTemplate-i18n
+    >
+        <ng-template pTemplate="header" styleClass="p-datatable-striped">
+            <tr>
+                <th style="width: 10%"></th>
+                <th style="width: 20%" pSortableColumn="word">
+                    <span i18n>Word</span>
+                    <p-sortIcon field="word"></p-sortIcon>
+                </th>
+                <th style="width: 20%" pSortableColumn="lemma">
+                    <span i18n>Lemma</span>
+                    <p-sortIcon field="lemma"></p-sortIcon>
+                </th>
+                <th style="width: 20%" pSortableColumn="type">
+                    <span i18n>Type</span>
+                    <p-sortIcon field="type"></p-sortIcon>
+                </th>
+                <th style="width: 10%;"pSortableColumn="samples">
+                    <span i18n>Samples</span>
+                    <p-sortIcon field="samples"></p-sortIcon>
+                </th>
+            </tr>
+        </ng-template>
+        <ng-template pTemplate="body" let-row let-expanded="expanded">
```

Comment:
Very minor issue: if you expand one result and then run a new query, the same node will stay expanded even though the content is different

Reply:
Good spot!